### ISSUE TITLE
Failed to launch debug adapter error with latest VS preview & .NET 8.0 runtime (from main)

### ISSUE THREAD

**Issue by mitchdenny** [dotnet/aspnetcore#48514]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

I was doing some experimentation with Blazor. I created a project using the new _Blazor Web App_ template and hit F5. I then launches edge to a blank page:

If I leave it I get this error (from VS - doesn't seem to be triggered by the app itself):

<img width="206" alt="Screenshot 2023-05-30 140444" src="https://github.com/dotnet/aspnetcore/assets/513398/19332bbf-c131-49f4-bf64-9a1d9b572404">

It is possible for me to browse to the Blazor app and it works, but eventually I get the error above after some kind of timeout.

### Expected Behavior

Should be able to launch Blazor Web App template using F5 with no problems.

### Steps To Reproduce

1. File | New Project (Blazor Web App)
2. Hit F5
3. Wait.

### .NET Version

8.0.100-preview.6.23279.4

### Anything else?

VS Version: Version 17.7.0 Preview 2.0 [33728.48.main]

**Comment by mitchdenny** [comment]
Some further information. I checked Blazor Web Assembly project in VS2022 w/ .NET 7.0 and it works fine. You can see it launch the browser to a blank page and then the automation kicks in and navigates it to the Blazor app.

**Comment by mitchdenny** [comment]
Just tested with Google Chrome, it doesn't work either (same issue). I also tested the Blazor Web Assembly app w/ .NET 7.0 in IntPreview and it managed to cause the browser to navigate, but the Blazor app itself crashed (yellow bar at the bottom of the window with Reload).

**Comment by mitchdenny** [comment]
So I can see requests going to to the ASP.NET app from the browser (```/_framework/debug/ws-proxy?...```) - and returning with a 404. So it seems like the debug proxy isn't working.

**Comment by mitchdenny** [comment]
OK, I think I've figured this out. I looked at the sample in `src/Components` and noticed that it was working. I then took a list of all the `node.exe` processes running on the machine to figure out which one was the dev proxy:

```Get-Process node | fl Id,CommandLine```

I then looked at the parent:

```(Get-Process -Id [theid]).Id```

Turns out the parent process for the dev server wasn't the ASP.NET process like I thought it would be, it was ```devenv.exe```. Which makes sense the ```DebugProxyLauncher``` doesn't seem to be getting used anymore. Anyway ... I had a closer look at ````launchSettings.json``` in the sample in the repo and it doesn't have the ```inspectUrl``` property - so I removed that from mine ... and bingo - it worked.

I think this means that we need to remove the inspectUrl from the Blazor App template in the repo?
